.github/workflows/nightlies.yml

```diff
@@ -0,0 +1,69 @@
+name: Nightlies
+on:
+  schedule:
+    - cron: "0 6 * * *"
+  push:
```

Comment:
This is based on https://github.com/freedomofpress/securedrop-client/pull/1960, which hasn't been merged yet.

Reply:
OK, so we expect to get both nightlies _and_ a new build everytime we see a push on `main`. The nightlies themselves are expected to trigger a double-build, because they themselves will kick off a push to `main`. Is that correct?

Reply:
> The nightlies themselves are expected to trigger a double-build, because they themselves will kick off a push to `main`. Is that correct?

No, because the nightlies just push to securedrop-yum-test's main and not securedrop-workstation's main.

Reply:
Or maybe I'm misinterpreting what you meant by "both nightlies _and_ a new build"

Reply:
No, you're right. The order of operations as I understand it would be:

1) Push occurs to `securedrop-workstation`
2) This triggers the `on: push` workflow here, causing a nightly build to be pushed to `securedrop-yum-test`
3) _That_ push kicks off the `on: push` cleanup job in `securedrop-yum-test` to preserve a maximum of 4 builds, which recursively triggers itself once with no expected side effect

And the same occurs on a nightly basis. Does that sound right?

Reply:
Yep!

Reply:
Excellent.The workflow LGTM, but I'll take another look on Monday for a formal review if nobody beats me to it. :)